src/ConfigEditor/ConfigEditor.tsx

```diff
@@ -84,6 +84,7 @@ export function ConfigEditor(props: Props) {
         });
       });
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
is this needed? seems like nothing is changing? maybe just fixing an old lint error?

Reply:
probably updated packages pulls in new eslint rules?